.github/auto-assignees.yaml

```diff
@@ -0,0 +1,30 @@
+# This assigns a PR to its author
+addAssignees: true
```

Comment:
this seems not required currently

Reply:
Why do you think so? Every example (even the upstream one) explicitly sets this to true:

https://github.com/kentaro-m/auto-assign-action#single-reviewers-list

And matches what `harbor` repo has set:
https://github.com/goharbor/harbor/blob/a826720cc94dd9ac8d3495ba74a913055a2a9268/.github/auto-assignees.yml#L3

Reply:
it should be either true or author, we need true so we have array of assignees

Reply:
please add wy65701436 , and assign to him

Reply:
Do you mean adding @wy65701436 to the list of assignees?

I took the current list from here: https://github.com/goharbor/harbor-operator/blob/master/MAINTAINERS

Reply:
@MarcelMue correct but Yan Wang @wy65701436 is currently the overall tech lead for Harbor so it's good to have him :)

Reply:
@wy65701436 can you confirm please

Reply:
Done.